### PR TITLE
Add gke-1.9 vs etcd 3.0.17 test suites

### PR DESCRIPTION
**Pull request by krzyzacy**
per discussion offline
xref https://github.com/kubernetes/test-infra/issues/6147
/cc @lookuptable @janetkuo @droot
/area jobs

/hold
until the version is available

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -5807,6 +5807,66 @@
       "generated"
     ]
   },
+  "ci-kubernetes-e2e-gke-cos-oldetcd-default": {
+    "args": [
+      "--check-leaked-resources",
+      "--deployment=gke",
+      "--env=TEST_ETCD_VERSION=3.0.17",
+      "--extract=v1.9.0-gke.0",
```
Comment (janetkuo) [inline jobs/config.json]: Use `v1.9.1-gke.0` instead?

Comment (krzyzacy) [reply jobs/config.json]: good catch, fixed

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -2022,6 +2022,13 @@ test_groups:
 - name: istio-e2e-cluster_wide-auth
   gcs_prefix: istio-prow/e2e-cluster_wide-auth
   num_failures_to_alert: 1
+# gke 1.9.0 vs old etcd
```
Comment (janetkuo) [inline testgrid/config/config.yaml]: nit: 1.9.1

Comment (krzyzacy) [reply testgrid/config/config.yaml]: done

**Comment by janetkuo** [comment]
We can remove the do-not-merge label now as the version is available now. Can someone else review this PR?

**Comment by krzyzacy** [comment]
/hold cancel
can some1 lgtm?

**Comment by krzyzacy** [comment]
/hold cancel
try to get some signal

**Pull request merged**